features/bevragen/persoon/verblijfplaats/locatie/in-onderzoek.feature

```diff
@@ -0,0 +1,192 @@
+# language: nl
+
+Functionaliteit: Persoon: verblijfplaats binnenland (locatie) velden zijn in onderzoek
+
```

Comment:
Ik mis hier de Rule:

`Rule: Het in onderzoek zijn van een veld en bijbehorende datumIngangOnderzoek worden geleverd als het betreffende veld wordt gevraagd`